metricbeat/module/kvm/dommemstat/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- name: dommemstat
```

Comment:
We should complete the fields.yml here and have a system tests that validates that all fields are documented.

Reply:
@amandahla could you complete this fields file?